labs/lab-10/tasks/overflow-for-binary/support/exploit.py

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: BSD-3-Clause
+import subprocess
+
+
+def run_executable():
+    argument = ""  # TODO: Put here the payload you have discovered
+    subprocess.run(["../support/overflow_in_binary", argument])
```

Comment:
```suggestion
    subprocess.run(["./overflow_in_binary", argument])
```
Run the tester script from this folder so hardcoding this path isn't needed. Otherwise it's weird for students running this exploit from `support/` to run it referring `../support/` which is `.`. Do this for all `exploit.py`s.